utils/download_sqlite_all.js

```diff
@@ -50,9 +51,19 @@ function download(callback) {
   const generateCommand = (sqlite, directory) => {
     let extract;
     if (/\.db\.bz2$/.test(sqlite.name_compressed)) {
-      extract = `bunzip2`;
-    } else if(/\.db\.tar\.bz2$/.test(sqlite.name_compressed)) {
-      extract = `tar -xjO`;
+      // Check if we have lbzip2 installed
+      if (commandExistsSync('lbzip2')) {
+        extract = 'lbzip';
```

Comment:
When testing this code I ran into an issue with this command. Shouldn't it be `lbzip2` here?